test/test_respondent_controller.py

```diff
@@ -1839,31 +1863,31 @@ def test_delete_respondent_mark_for_deletion(self):
         with self.assertRaises(Exception):
             self.get_respondent_by_id(respondent.party_uuid)
 
-    def test_batch_delete_user_data_marked_for_deletion(self):
-        def mock_put_iac(*args, **kwargs):
-            return MockResponse('{"active": false}')
-
-        self.mock_requests.put = mock_put_iac
-        self.populate_with_respondent(respondent=self.mock_respondent_with_id)
-        self.populate_with_business()
-        self.associate_business_and_respondent(
-            business_id=DEFAULT_BUSINESS_UUID, respondent_id=self.mock_respondent_with_id["id"]
-        )
-        self.populate_with_enrolment()
-        respondent = respondents()[0]
-        self.assertEqual(respondent.mark_for_deletion, False)
-        respondent_controller.update_respondent_mark_for_deletion(respondent.email_address)
-        response_respondent = self.get_respondent_by_id(respondent.party_uuid)
-        self.assertEqual(response_respondent["markForDeletion"], True)
-        respondent_1 = MockRespondent()
-        respondent_1.attributes(emailAddress="res1@example.com", mark_for_deletion=True)
-        respondent_1 = self.populate_with_respondent(respondent=respondent_1.as_respondent())
-        response = self.delete_user_data_marked_for_deletion()
-        self.assertStatus(response, 204)
-        with self.assertRaises(Exception):
-            self.get_respondent_by_id(respondent.party_uuid)
-        with self.assertRaises(Exception):
-            self.get_respondent_by_email(respondent_1.email_address)
+    # def test_batch_delete_user_data_marked_for_deletion(self):
```

Comment:
Why has this been commented out? There should be reason in the code, on the PR. Or if not needed just removed